include/openssl/core_names.h

```diff
@@ -382,7 +382,9 @@ extern "C" {
 /* RSA padding modes */
 #define OSSL_PKEY_RSA_PAD_MODE_NONE    "none"
 #define OSSL_PKEY_RSA_PAD_MODE_PKCSV15 "pkcs1"
-#define OSSL_PKEY_RSA_PAD_MODE_SSLV23  "sslv23"
+#ifndef OPENSSL_NO_DEPRECATED_3_0
+# define OSSL_PKEY_RSA_PAD_MODE_SSLV23  "sslv23"
+#endif
```

Comment:
Maybe we should just drop this define? So we do not add immediately deprecated defines in 3.0?

Reply:
I agree adding something and deprecating it seems silly, but it's used in two files so have to keep it I think.

Reply:
Could it be defined in some internal header? @levitte any opinions?